msautotest/wxs/wfs_query_cache.map

```diff
@@ -8,6 +8,10 @@
 
 # RUN_PARMS: wfs_query_cache.json [MAPSERV] QUERY_STRING="map=[MAPFILE]&SERVICE=WFS&VERSION=1.0.0&REQUEST=GetFeature&TYPENAME=province&MAXFEATURES=10&OUTPUTFORMAT=geojson" > [RESULT]
 
+# RUN_PARMS: wfs2_query_cache.json [MAPSERV] "QUERY_STRING=map=[MAPFILE]&SERVICE=WFS&VERSION=2.0.0&REQUEST=GetFeature&TYPENAMES=province&MAXFEATURES=10&OUTPUTFORMAT=geojson" > [RESULT]
+
+# RUN_PARMS: wfs2_query_cache_no_default_extent.json [MAPSERV] "QUERY_STRING=map=[MAPFILE]&SERVICE=WFS&VERSION=2.0.0&REQUEST=GetFeature&TYPENAMES=province2&MAXFEATURES=10&OUTPUTFORMAT=geojson" > [RESULT]
```

Comment:
is this test + added layer really needed with respect to what you changed ? (just for the sake of minimizing things)

Reply:
I was worried setting `"wfs_use_default_extent_for_getfeature" "false"` might have skipped the `msQueryByRect` function, and then there could be features in the cache which weren't reprojected. This is not the case as a `MS_INIT_INVALID_RECT` is set and used in the `msQueryByRect`, but thought it might be good to keep this test in case code is refactored in the future.

Reply:
ok, fair point